src/app/profile/tenant/tenant.component.less

```diff
@@ -0,0 +1,84 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
```

Comment:
none of this is encapsulated, so you are running the risk of targeting pieces across the site.